examples/src/main/cc/wfa/examples/streaming/hello_streaming_client.cc

```diff
@@ -1,109 +1,76 @@
-// Copyright 2021 The Cross-Media Measurement Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#include "src/main/cc/wfa/examples/streaming/hello_streaming_client.h"
-
+/*
+ * Copyright 2021 The Cross-Media Measurement Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <grpc/grpc.h>
+#include <grpcpp/channel.h>
+#include <grpcpp/client_context.h>
 #include <grpcpp/create_channel.h>
 #include <grpcpp/security/credentials.h>
-
-#include <iostream>
-
-#include "absl/container/fixed_array.h"
-
-namespace wfa_examples {
-namespace streaming {
-namespace {
-
-using ::grpc::Channel;
-using ::grpc::ClientContext;
-using ::grpc::ClientWriter;
-using ::grpc::CreateChannel;
-
-absl::Status MakeStatus(const grpc::Status& grpc_status) {
-  if (grpc_status.ok()) {
-    return absl::OkStatus();
-  }
-
-  auto status_code = static_cast<absl::StatusCode>(grpc_status.error_code());
-  return absl::Status(status_code, grpc_status.error_message());
+#include "src/main/proto/wfa/examples/streaming/hello_streaming_service.grpc.pb.h"
+
+using grpc::Channel;
+using grpc::ClientContext;
+using grpc::ClientReader;
+using grpc::ClientReaderWriter;
+using grpc::ClientWriter;
+using grpc::Status;
+using wfa_examples::streaming::HelloStreaming;
+using wfa_examples::streaming::SayHelloStreamingRequest;
+using wfa_examples::streaming::SayHelloStreamingResponse;
+
+std::string GetServerAddress() {
```

Comment:
This seems unnecessary, maybe just write `std::string server_address = "0.0.0.0:50051";` in main.